src/SnapKeyring.test.ts

```diff
@@ -1128,18 +1163,22 @@ describe('SnapKeyring', () => {
 
     it('removes an account', async () => {
       mockSnapController.handleRequest.mockResolvedValue(null);
-      await keyring.removeAccount(accounts[0].address);
+      await keyring.removeAccount(ethEoaAccount1.address);
       expect(await keyring.getAccounts()).toStrictEqual([
-        accounts[1].address.toLowerCase(),
+        ethEoaAccount2.address.toLowerCase(),
```

Comment:
No need for `.toLowerCase` here, the address has already been lowered during declarations, this is also true for the `accounts[2]` and `accounts[3]`.

Unless this was intentional to have this explicit?

Reply:
Yea this can be removed, but in run time if we were to compare the addresses, we should do a case insensitive comparison because the snap could return a checksum/nonchecksum address.